game.py

```diff
@@ -0,0 +1,359 @@
+    dialogue = Dialogue()
+    dialogue.show_dialogue("hero_awakening")
+    print_with_delay("you grip the weapon tighter, felling its weight-heavy.")
+    print("\n")
+    dialogue.show_dialogue("robed_explanation")
+   
+    choose_hero()
+
+def choose_hero():
+    #print("\n1. Shield Hero - The unbreakableGuardia, protector of the realm")
+    print("\n1. Sword Hero - The fearless warrior, cutting dowmn enemies with raw strength.")
+    print("2. Spear Hero - The swift sticker, delivering precise amnd devastating blows.")
+    print("3. Bow Hero - The unseen hunter, eleimenting threats from afar.")
+   
+    hero_classes = {
+
+
+        #"1": (ShieldHero, "Shield Hero"),
+        "1": (SwordHero, "Sword Hero"),
+        "2": (SpearHero, "Spear Hero"),
+        "3": (BowHero, "Bow Hero")
+        }
+    hero_lore ={
+        #"1": (
+           # "** The Shield hero **\n"
+            #"A bastioon  of unyiedling resolve, the Shield Hero stand as the last line of defense."
+            #"when all others fall,, they remain, shieldin the reaml form annihilation"
+        #),
+        "1": (
+            " ** The Swoar Hero**\n"
+            "A warrior of unparalleded skill, the Sword eor craves through the darkness with each swing."
+            "feared by enemies, revered by allies, they are the embbodiement of raw strength and unrelenting will"
+        ),
+        "2": (
+            "** The Spear Hero** \n"
+            "A MAster of speed and precsion, the Spear Hero brak threough enemy line s with lightning-fast strikes"
+        ),
+        "3":  (
+            "** The Bow Hero **\n"
+            "A phantom in the shadoe the Bow hero eliminates threats befor they even realiz thier doom."
+            "Silent, deadly, and alwasy unseen, they are unseen force that turn the tide of war"
+            )
+           
+    }
+    print("\n Previewing heroes...\n")
+    for num, (HeroClass, Hero_title) in hero_classes.items():
+        print_with_delay(hero_lore[num])
+        temp_hero = HeroClass("Preview")
+        temp_hero.show_stats()
+        print("-" * 40)
+    while True:
+        choice = input("\nWhich hero do you choose? (enter 1-4): ")
+        if choice in hero_classes:
+            HeroClass, Hero_title = hero_classes[choice]  
+            print_with_delay(f"\nYou have chosen: **{Hero_title}!**")
+            hero_name = input("\nWhat is your name, hero? ")
+
+            hero =HeroClass(hero_name)
+           
+            print_with_delay(f"\nYou have aquired the **{hero.weapon.name}!**")
+
+            while True:
+                command = input("Press 'I' to opnen yur inventoery or 'C' to continue: ").strip().lower()
+                if command == "i":
+                    hero.open_inventory()
+                elif command == 'c':
+                    print("Continuing the game...")
+                    break
+                else:
+                    print("Invalid input, Press 'I' fro inventory or 'c' to continue.")
```

Comment:
Typo and Consistency 

```suggestion
                    print("Invalid input, Press 'I' for inventory or 'C' to continue.")
```